### PR TITLE
Fix #447 v-key issue

### LINKED ISSUE #447: Pressing V Crashes
Pressing `v` kills the renderer:

```
~/.local/lib/python3.7/site-packages/pyvista/plotting/plotting.py in <lambda>()
    229         b_left_down_callback = lambda: self.iren.AddObserver('LeftButtonPressEvent', self.left_button_down)
    230         self.add_key_event('b', b_left_down_callback)
--> 231         self.add_key_event('v', lambda: self.isometric_view_interactive())
    232 
    233 

~/.local/lib/python3.7/site-packages/pyvista/plotting/plotting.py in isometric_view_interactive(self)
    420         interactor = self.iren.GetInteractorStyle()
    421         renderer = interactor.GetCurrentRenderer()
--> 422         renderer.view_isometric()
    423 
    424     def update(self, stime=1, force_redraw=True):

AttributeError: 'NoneType' object has no attribute 'view_isometric'
```

### PR DESCRIPTION
**Pull request by banesullivan**
Fix #447 by grabbing the last active renderer if the current interactive renderer is `None`

### CODE REVIEW

File: pyvista/plotting/plotting.py [hunk 1]
```diff
@@ -1796,7 +1798,7 @@ def remove_actor(self, actor, reset_camera=False):
     def add_actor(self, uinput, reset_camera=False, name=None, loc=None,
                   culling=False, pickable=True):
         """Add an actor to render window.
-
```
Comment (banesullivan) [inline pyvista/plotting/plotting.py]: @GuillaumeFavelier, can we trailing whitespace to a pycodestyle check?

Comment (GuillaumeFavelier) [reply pyvista/plotting/plotting.py]: We might need more than just `pydocstyle` for that: http://pycodestyle.pycqa.org/en/latest/intro.html#id3

Comment (banesullivan) [reply pyvista/plotting/plotting.py]: Hm. Then maybe we should just ask all contributors to turn on a setting in their text editor to strip trailing whitespace? For atom, [this works](https://atom.io/packages/whitespace):

Screen Shot 2019-11-26 at 10 34 58 AM

**Comment by banesullivan** [comment]
I was able to reproduce the error locally and this fixed it for me

**Pull request merged**